experiments/integrated/fluxnet/fluxnet_domain.jl

```diff
@@ -0,0 +1,18 @@
+# Domain setup
+# For soil column
+nelements = 10
+zmin = FT(-2)
+zmax = FT(0)
+dz_bottom = FT(0.5)
+dz_top = FT(0.025)
+
+land_domain = Column(;
+    zlim = (zmin, zmax),
+    nelements = nelements,
+    dz_tuple = (dz_bottom, dz_top),
+)
+canopy_domain = ClimaLSM.Domains.obtain_surface_domain(land_domain)
+
+# Number of stem and leaf compartments. Leaf compartments are stacked on top of stem compartments
+compartment_midpoints = [h_stem / 2, h_stem + h_leaf / 2]
```

Comment:
I believe these should be specific to the site also. Im not sure if the Vaira site (a grassland site, n_stem = 0) will run with 2 compartments, because n_stem+n_leaf =1.

It's possible that z_min is site specific too, though that is probably less important?

Reply:
Perhaps we could have another file in the different site directories, "simulation_setup.jl" which specifies some domain stuff, and dt?